packages/nimble-components/src/anchor/styles.ts

```diff
@@ -0,0 +1,37 @@
+import { css } from '@microsoft/fast-element';
+import { bodyDisabledFontColor } from '../theme-provider/design-tokens';
+import { focusVisible } from '../utilities/style/focus';
+import { linkStyles, linkColors } from '../patterns/link/styles';
+
+export const styles = css`
```

Comment:
In the theme matrix story, I was mildly surprised that the disabled and prominent link looked identical to the default disabled link. I guess I was expecting the disabled prominent version to be a faded green rather than grey. Not my call, but could be worth running it past Brandon if he didn't already provide an opinion. If this is following established patterns then that takes precedence over my opinion too.